lms/djangoapps/philu_api/views.py

```diff
@@ -259,8 +259,11 @@ def send_alquity_fake_confirmation_email(request):
     return JsonResponse({'success': success})
 
 
+@csrf_exempt
 def resend_activation_email(request):
-    user = request.user
+    data = json.loads(request.body)
```

Comment:
Let's please use request.POST and also let's please add require_post decorator because user id is in data.

Reply:
I have added the said decorator. Since I am sending the data as JSON from front-end, will have have use request.body, instead of request.POST